tests/test-components/FileItem.spec.tsx

```diff
@@ -18,7 +22,8 @@ describe('FileItem', () => {
     onDoubleClick: () => {},
     selected: false,
     selectFile: () => {},
-    style: {}
+    style: {},
+    trans: trans
```

Comment:
You don't need to change it. But FYI when passing a variable named as the key, you can simply write:

```
style: {},
trans
```